### PR TITLE
fix(feedback): add feedback shimming to event manager

### PR DESCRIPTION
**Pull request by JoshFerge**
If we determine it is okay to do so, we shim user reports to user feedback in the event_manager. 

depends on https://github.com/getsentry/sentry/pull/70116

### CODE REVIEW

File: src/sentry/event_manager.py [hunk 1]
```diff
@@ -2852,10 +2854,23 @@ def _update_user_reports_with_event_link(job: Job, group_info: GroupInfo) -> Non
...
+            {
+                "name": report.name,
+                "email": report.email,
+                "comments": report.comments,
+                "event_id": report.event_id,
+                "level": "error",
+            },
+            event,
+            project,
+            FeedbackCreationSource.USER_REPORT_ENVELOPE,
```
Comment (aliu3ntry) [inline src/sentry/event_manager.py:2868]: Maybe change this source to update_user_reports?

File: src/sentry/event_manager.py [hunk 2]
```diff
@@ -2852,10 +2854,23 @@ def _update_user_reports_with_event_link(job: Job, group_info: GroupInfo) -> Non
...
+        )
+
     user_reports_updated = user_reports_without_group.update(
```
Comment (aliu3ntry) [inline src/sentry/event_manager.py:2871]: Nit
```suggestion
    num_user_reports_updated = user_reports_without_group.update(
```

File: src/sentry/event_manager.py (hunk 1 above)
Comment (JoshFerge) [reply src/sentry/event_manager.py:2868]: why?

Comment (JoshFerge) [reply src/sentry/event_manager.py:2868]: (we can't tell the source origin of ths feedback, so envelope generally works and we have frontend logic for that)

Comment (aliu3ntry) [reply src/sentry/event_manager.py:2868]: Got it. My reasoning was this "shimming" is sorta taking the place of the update_user_reports task.. but I guess we should remove that source type if/when the task isn't needed

**Review by cmanallen — APPROVED** [review]

**Comment by JoshFerge** [comment]
closing as we're gonna move this whole flow to post process instead. PR incoming

**Pull request closed without merging**